README.md

```diff
@@ -42,8 +42,9 @@ foreman-installer --scenario                     "katello"\
                   --foreman-proxy-puppetca       "false"
 
 # Generate certificates for installing Foreman proxy on another system
-foreman-proxy-certs-generate --foreman-proxy-fqdn "myforeman-proxy-content.example.com"\
-                                     --certs-tar    ~/myforeman-proxy-content.example.com-certs.tar
+katello-generate-cert-archive --server-fqdn "myforeman-proxy-content.example.com"\
```

Comment:
Could we drop the katello naming aspect?

Reply:
So just ‘generate-cert-archive’ ?

I think the katello name is important as it’s explicit for what it is for. It’s also consistent with other tools (backup, restore, service, etc... )

Reply:
How about `foreman-generate-cert-archive` ?

Reply:
Does the Foreman name make sense if the certs archive cannot be used with vanilla foreman? If there is a plan to make it compatiable with Foreman then, I'd be happier naming it that. 

I think the current naming can make the two "brands" (Katello & Foreman) confusing to users, sometimes we swap the names around for no good reason. To me at least the Foreman name implies I would use it with Foreman, but in this case the tool cannot be used with vanilla Foreman.